### PR TITLE
Fix assorted typos and formatting variations in programs.md.

### PR DESCRIPTION
**Pull request by atsampson**
Some of the choices I've made here are a bit arbitrary - e.g. ARPAnet vs. Arpanet or nickname vs. nicname, where both were used - in these cases, I've tried to pick what seemed to be the most common spelling in the docs/mail archives we've got.

### CODE REVIEW

File: doc/programs.md [hunk 1]
```diff
@@ -73,9 +73,9 @@
 - FILE, Chaosnet file server.
 - FIND, search for files.
 - FRETTY, display list of free TTYs.
-- FTPS, FTP Server.
-- FTPU, FTP Client.
-- GCMAIL, delete old files from .MAIL.
+- FTPS, FTP server.
+- FTPU, FTP client.
+- GCMAIL, delete old files from .MAIL.;.
```
Comment (larsbrinkhoff) [inline doc/programs.md]: I don't think `;` is needed unless it's a full file name with FN1 etc.  But I haven't checked all files to see what the most common usage is.

File: doc/programs.md [hunk 2]
```diff
@@ -164,45 +164,45 @@
...
+- SCANDL, TTY output spy.
 - SCHEME, Scheme interpreter.
 - SCNV, convert plotter files to XGP scan files.
-- SCRAM, encypt/decrypt file.
+- SCRAM, encrypt/decrypt file.
 - SCRMBL/UNSCR, scramble/unscramble file.
 - SEND, REPLY, replacements for DDT :SEND.
 - SENDS, Chaosnet SEND server.
-- SENSOR, sends censor.
+- SENSOR, SEND's censor.
```
Comment (larsbrinkhoff) [inline doc/programs.md]: I think "sends" is a plural form here.

File: doc/programs.md (hunk 1 above)
Comment (atsampson) [reply doc/programs.md]: I was going for consistency with TMPKIL ("clean out old files in .TEMP.;."), but equally we could remove the semicolon from the latter...

File: doc/programs.md (hunk 2 above)
Comment (atsampson) [reply doc/programs.md]: I was thinking of it as censoring SEND, but yes, I guess it works both ways!

How about "An alternate message receiver", which is how it's described in DIR?

File: doc/programs.md [hunk 3]
```diff
@@ -164,45 +164,45 @@
...
+- STYLOG, convert PTY output file into ASCII file.
 - SUPDUP, Supdup client.
 - SYSCHK, check up on system job.
 - SYSLOD, system load histogram.
 - SYSMSG, displays system messages.
 - RABBIT, Scheme compiler.
 - TAGS, generate tags table for sources.
 - TALK/WHO/WHOJ/WHOM/USERS, list users.
-- TCTYP and CRTSTY, terminal handling.
+- TCTYP/CRTSTY, terminal handling.
```
Comment (larsbrinkhoff) [inline doc/programs.md]: TCTYP and CRTSTY should probably be separate lines.

File: doc/programs.md (hunk 2 above)
Comment (larsbrinkhoff) [reply doc/programs.md]: Yes, that is more descriptive.

**Pull request merged**